### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://nikushaSheremet.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by Andrii-Stiebalov — APPROVED** [review]
Good job!

File: src/styles/main.scss [hunk 1]
```diff
@@ -11,4 +11,45 @@ body {
...
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+
+  margin: 0 auto ;
+
+  @media (max-width: 488px) {
+    & {
+      grid-template-columns: repeat(1, 200px);
```
Comment (AndriiRut) [inline src/styles/main.scss:24]:
```suggestion
      grid-template-columns: repeat(1, 200px); // this is default value. you can delete this
```

**Review by AndriiRut — CHANGES_REQUESTED** [review]
Good job!

File: src/styles/main.scss (hunk 1 above)
Comment (nikushaSheremet) [reply src/styles/main.scss]: fixed

**Review by AndriiRut — APPROVED** [review]
🔥

**Pull request closed without merging**